addons/resource.language.en_gb/resources/strings.po

```diff
@@ -5618,7 +5618,17 @@ msgctxt "#12022"
 msgid "Resume from {0:s}"
 msgstr ""
 
-#empty strings from id 12023 to 12309
+#empty strings from id 12023 to 12307
+
+#: xbmc/dialogs/GUIDialogKeyboardGeneric.cpp
```

Comment:
could you please also add a verbal description explaining purpose and context of the new string to help the translators?